inc/dashboard/namespace.php

```diff
@@ -26,6 +26,8 @@ function setup() {
 
 	add_action( 'load-index.php', __NAMESPACE__ . '\\load_dashboard' );
 	add_action( 'admin_menu', __NAMESPACE__ . '\\add_widgets_submenu' );
+	add_action( 'admin_notices', __NAMESPACE__ . '\\add_notices_wrapper_open', 0 );
+	add_action( 'admin_notices', __NAMESPACE__ . '\\add_notices_wrapper_close', 999 );
```

Comment:
We could probably go higher on the closing div priority, but it's kind of hit and miss whether someone will use a reasonable value though. Lets see how we go.

In answer to your question about only showing the wrapper on the Altis dashboard page we do need to do that. You can place the `add_action()` calls inside the `load_dashboard()` function after the early returns.